iModelJsNodeAddon/GeoServicesInterop.cpp

```diff
@@ -67,4 +67,54 @@ BentleyStatus GeoServicesInterop::GetGeographicCRSInterpretation(BeJsValue resul
     return (BentleyStatus)status;
     }
 
+//---------------------------------------------------------------------------------------
+// @bsimethod
+// @param ignoreLegacy If true, only return GCS's that are not considered legacy.
+// @param extent If provided, only return GCS's that contain the given extent. Minimum longitude and latitude correspond to extent.low.x and extent.low.y, respectively.
+// Maximum longitude and latitude correspond to extent.high.x and extent.high.y, respectively.
+//---------------------------------------------------------------------------------------
+bvector<GCSListResponseProps> GeoServicesInterop::GetListOfGCS(bool ignoreLegacy, DRange2dCP extent)
+    {
+    static GeoCoordinates::BaseGCSPtr ll84GcsPtr = GeoCoordinates::BaseGCS::CreateGCS("LL84");

```

Comment:
ll84GcsPtr  is unused, you can remove this since we aren't checking for DatumConversion to ll84, like Conceptstation would

Reply:
Done